### DISCUSSION TITLE
Rook-ceph-mds memory consumption is very high

### DISCUSSION THREAD

**Discussion by ahmedTouati** [rook/rook#9504] [Q&A]
we have a cluster-on-pvc installation of Rook with 3 OSDs but recently the memory consumption of both rook-ceph-mds-a and rook-ceph-mds-b became very high (16go) which caused these two pods to always restart because of OOM.
Note: we didn't set any limits on mds memory.

**Comment by leseb** [comment]
How does the workload look like in this environment? I wouldn't call 16GB an abnormal consumption for a metadata server unless it's doing nothing.
